pkg/peeker/reader.go

```diff
@@ -47,22 +47,14 @@ func (r *Reader) fill(min int) error {
 	r.buffer = r.buffer[:cap(r.buffer)]
 	copy(r.buffer, r.cursor)
 	clen := len(r.cursor)
-	space := len(r.buffer) - clen
-	for space > 0 {
-		cc, err := r.Reader.Read(r.buffer[clen:])
-		if cc > 0 {
-			clen += cc
-			space -= cc
-		}
-		if err != nil {
-			if err == io.EOF {
-				r.eof = true
-				break
-			}
+	cc, err := io.ReadAtLeast(r.Reader, r.buffer[clen:], min)
```

Comment:
I like how this is shorter.

Honestly curious: Is there a mnemonic (or something) else behind `cc`? (`io.ReadAtLeast` calls  it `n`.)

Reply:
@mccanne started it

Reply:
Then call it `n`!